spec/models/rent_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe Rent do
+  subject(:rent) { build(:rent) }
+
+  it { is_expected.to validate_presence_of(:start_date) }
+  it { is_expected.to validate_presence_of(:end_date) }
+
+  describe '#save' do
+    context 'With valid params' do
+      it 'returns the new rent object' do
+        rent.save!
+        expect(rent).to be_present
+      end
+    end
+
+    let(:emptyRent) { Rent.new }
```

Comment:
You may check properties one at a time, like `build( :rent, book: null)` and check for that instead.